docs/docs.html

```diff
@@ -631,6 +632,14 @@ <h5 id="config-details">Config Options</h5>
 									A callback function, identical to step, which activates streaming. However, this function is executed after every <i>chunk</i> of the file is loaded and parsed rather than every row. Works only with local and remote files. Do not use both chunk and step callbacks together. For the function signature, see the documentation for the step function.
 								</td>
 							</tr>
+							<tr>
+								<td>
+									<code>chunkSize</code>
+								</td>
+								<td>
+									Overrides <code>Papa.LocalChunkSize</code> and <code>Papa.RemoteChunkSize</code>.
```

Comment:
I think we should explain what do both values instead of just saing that it overrides the behaviour.

Reply:
Ah I didn't do that because I thought they were already explained at the end of the document. Do you want me to link them to https://www.papaparse.com/docs#configurable or should I explain them here again?

Reply:
Yes please, just add the link.

Reply:
Added 😄